scriptcs

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+## Make sure to chmod +x this file!
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+mono "$DIR/scriptcs.exe" $@
```

Comment:
I think this should probably live under `/src/ScriptCs` with `Copy to Output Directory` set to `Copy Always`, so it just gets emitted into `bin/Release` along with everything else.